### DISCUSSION TITLE
Which flavor of Regex is this plugin using?

### DISCUSSION THREAD

**Discussion by chrisgrieser** [jglev/obsidian-apply-patterns-plugin#13] [Q&A]
I noticed some minor things, like e.g. that I cannot use "\n" in the replacement, even though I can at Regex101. 

I assume I used a different flavor at Regex101 than this plugin uses?

**Comment by jglev** [comment]
The plugin uses the ECMAScript / Javascript flavor. The lack of `\n` support in `To` strings was a bug that I hadn't realized (`\n` seems to be working in `From` strings for me -- is it for you, as well?). I _think_ that it's fixed [here](https://github.com/publicus/obsidian-apply-patterns-plugin/commit/87d7e0d27ade59fcbd885d2915eb00f91758f6a5), set for inclusion in the next version of the plugin. Thank you for bringing it up!

Your question made me realize that it would be useful to document this, ha, so I've added it just now for the next version, [here](https://github.com/publicus/obsidian-apply-patterns-plugin/commit/9865c22cd7e691776a1eb885f8d194d5ef232638).

**Reply by chrisgrieser** [reply]
ah, I see. great, thanks for fixing!

Wasn't that much of a big deal, as the multiline-TO-field allowsed to manually enter line breaks, but for copypasting from sites like regex101 it's nonetheless relevant. 👍️

**Reply by FelipeRearden** [reply]
Thanks @chrisgrieser for pointing this! I thought it was an Obsidian issue. 

I'm glad to know that `\n` is now working. Thanks @publicus for this feature!!!!!

**Reply by FelipeRearden** [reply]
In my opinion, this feature `\n` is working perfect in the version 1.2.0 running in Obsidian 0.12.10 on MacOS

Thank you very much @publicus !!!!
